src/pages/SignIn.tsx

```diff
@@ -38,117 +51,139 @@ export default function SignIn() {
       <ThemeProvider theme={defaultTheme}>
         <Container component="main" maxWidth="xs">
           <CssBaseline />
+          <Typography
+            sx={{
+              marginTop: "90px",
+              textAlign: "center",
+              color: "#27963C",
+              fontFamily: 'cursive',
```

Comment:
You need to import the style from googlefonts in index.html. Also use of fallbacks
```suggestion
              fontFamily: 'Righteous', cursive, sans-serif, 
```